ppr-ui/tests/unit/test-data/mock-unit-notes.ts

```diff
@@ -55,20 +55,16 @@ export const mockUnitNotes: Array<UnitNoteIF> = [
     destroyed: false
   },
   {
-    documentType: UnitNoteDocTypes.EXTENSION_TO_NOTICE_OF_CAUTION,
-    documentId: '2',
-    documentRegistrationNumber: '789012',
-    documentDescription: 'Extension to Notice of Caution',
-    createDateTime: '2023-05-31T10:00:00Z',
-    effectiveDateTime: '2023-07-01T00:00:00Z',
-    expiryDateTime: '2023-07-31T23:59:59Z',
-    remarks: 'This is an extension to the notice of caution.',
+    documentType: UnitNoteDocTypes.RESTRAINING_ORDER,
+    documentId: '1',
+    documentRegistrationNumber: '123456',
+    documentDescription: 'Notice of Caution',
+    createDateTime: '2023-10-30T09:00:00Z',
+    effectiveDateTime: '2023-11-01T12:00:00Z',
+    expiryDateTime: '2023-11-30T23:59:59Z',
```

Comment:
Is the date changes to simulate future expiry dates? Then we need to make it a dynamic variable, to ensure the tests don't fail when the date passes.

Reply:
I changed the date to mimic how we receive the data from the API, where it is sorted by descending Creation Date.